app/src/main/java/swati4star/createpdf/fragment/SettingsFragment.java

```diff
@@ -134,39 +132,9 @@ public void onItemClick(int position) {
             case 5:
                 showImageScaleTypeDialog(mActivity, true);
                 break;
-            case 6:
-                changeMasterPassword();
-                break;
         }
     }
 
-    /**
-     * To modify master password of PDFs
```

Comment:
why is this function removed?

Reply:
i guess that  is in case that i did git push and something went wrong, because i did not make change in settings fragment, i will take a look at that today